docusaurus/docs/develop/path/2_environment.md

```diff
@@ -43,33 +45,47 @@ git clone https://github.com/buildwithgrove/path.git
 cd ./path
 ```
 
-### 2. Install Required Tools
+### 2. Install Required Tools (Linux & MacOS)
 
-:::tip
+#### Install `pocketd` CLI
 
-To install all dependencies automatically on a Linux system:
+To install the [`pocketd` CLI](https://dev.poktroll.com/category/pocketd-cli) on Linux or MacOS, run:
 
 ```bash
-make install_deps
+curl -sSL https://raw.githubusercontent.com/pokt-network/poktroll/main/scripts/install.sh | bash
 ```
 
-:::
+#### Install Required Local Tools
 
-The following tools are required to start a local PATH instance:
+To install all required local development tools on Linux or MacOS, run:
 
-**Local Deployment Tools:**
+```bash
+make install_tools
+```
+
+The above script will install the following tools which are required to start a PATH instance locally in development mode:
```

Comment:
Not a blocker: considering the number of dependencies and tools, providing a container image (along with commands that use a container based on that image) could significantly simplify the development process.

Reply:
I agree and I will add a TODO to handle this shortly.